src/sentry/web/frontend/base.py

```diff
@@ -509,26 +511,58 @@ def needs_sso(self, request: Request, organization: Organization) -> bool:
             return True
         return False
 
+    def _lookup_orm_org(self) -> Organization | None:
+        """
+        Used by convert_args to convert the hybrid cloud safe active_organization object into an org ORM.
+        This should really only be used by the Region or Monolith silo modes -- calling this in a Control silo
+        endpoint or codepath will result in exceptions.
+        :return:
+        """
+        organization: Organization | None = None
+        if self.active_organization:
+            try:
+                organization = Organization.objects.get(id=self.active_organization.organization.id)
+            except Organization.DoesNotExist:
+                pass
+        return organization
```

Comment:
Would this cause Mypy problems?
```suggestion
        if self.active_organization:
            try:
                return Organization.objects.get(id=self.active_organization.organization.id)
            except Organization.DoesNotExist:
                return None
```

Reply:
Probably not, that's something that could be addressed.  I think there might have been some other logic at one point that got paired down over time.  I often, wish the `get` method just return `Optional` minus the exception-as-control-flow here.